PF/Assignment1/PassengerTrain.java

```diff
@@ -0,0 +1,60 @@
+public class PassengerTrain {
+
+	int trainNumber;
```

Comment:
ACTION: You have to use oops inheritance concept here. Create a Parent class for Train having properties
trainNumber, startingPoint, endPoint, price. and create two child class PassengerTrain and GoodsTrain.